process/project_proposals.adoc

```diff
@@ -30,33 +30,36 @@ See the https://github.com/cncf/toc/blob/main/process/sandbox.md[Sandbox guideli
 
 === Incubation process
 
+Note: We have https://docs.google.com/presentation/d/1J9nti4JdiwLHxY15KtkmqyfP4OgNfrLAd3vxPvFTzsc/edit?usp=sharing[streamlined the Incubation process].
```

Comment:
@lizrice can we please copy the final slide diagram here for the record (but leave the note to the larger presentation for context)?

![image](https://user-images.githubusercontent.com/23304/115442288-d6e45800-a1df-11eb-8d2e-5b53c3086d0a.png)

Reply:
I _think_ I've already done this by [updating the png file](https://github.com/cncf/toc/pull/640/files#diff-bd9699ebc65945086aa74a13540fceaaf9519b07b6641f9fe31fa6d652fbc8c0). (Maybe it's not showing up in the doc preview because it's pulling the image from the main branch?)

Reply:
Yeah, the PR is referencing an image that is in the main branch. Once you merge the PR it should update the image. 

To avoid this, some people actually use [this method](https://gist.github.com/vinkla/dca76249ba6b73c5dd66a4e986df4c8d) to upload images in repos, so the image shows regardless of what branch you are on.  I believe that's what @dims suggested with the link he provided.

Reply:
ah Apologies!

Reply:
LGTM